test/xpu/run_test_with_only.py

```diff
@@ -45,13 +56,82 @@ def launch_test(test_case, skip_list=None, exe_list=None):
+    "test_comprehensive_nn_functional_unfold_xpu_float16",
+    "test_comprehensive_unfold_xpu_complex128",
+    "test_comprehensive_unfold_xpu_complex64",
+    "test_comprehensive_unfold_copy_xpu_float32",
+    "test_comprehensive_unfold_copy_xpu_float64",
+    "test_comprehensive_unfold_copy_xpu_complex32",
+    "test_comprehensive_unfold_xpu_float32",
+    "test_comprehensive_unfold_xpu_float64",
+    "test_comprehensive_unfold_copy_xpu_bool",
+    "test_comprehensive_unfold_copy_xpu_int16",
+    "test_comprehensive_unfold_copy_xpu_int32",
+    "test_quick_nn_functional_unfold_xpu_complex128",
+    "test_comprehensive_unfold_copy_xpu_int64",
+    "test_comprehensive_unfold_copy_xpu_int8",
+    "test_comprehensive_unfold_copy_xpu_uint8",
+    "test_quick_nn_functional_unfold_xpu_complex64",
+    "test_quick_unfold_xpu_complex32",
+    "test_quick_unfold_xpu_complex128",
+    "test_quick_unfold_copy_xpu_complex64",
+    "test_quick_unfold_copy_xpu_complex32",
+    "test_quick_unfold_copy_xpu_complex128",
+    "test_comprehensive_unfold_xpu_complex32",
+    "test_quick_unfold_xpu_complex64",
+    "test_quick_nn_functional_unfold_xpu_float64",
+    "test_quick_nn_functional_unfold_xpu_float32",
+    "test_quick_unfold_copy_xpu_float64",
+    "test_quick_unfold_copy_xpu_float32",
+    "test_quick_unfold_xpu_float64",
+    "test_quick_unfold_xpu_float32",
+    "test_comprehensive_unfold_xpu_int16",
+    "test_comprehensive_unfold_copy_xpu_bfloat16",
+    "test_comprehensive_unfold_xpu_int32",
+    "test_comprehensive_unfold_xpu_int64",
+    "test_comprehensive_unfold_xpu_int8",
+    "test_comprehensive_unfold_xpu_uint8",
+    "test_quick_unfold_copy_xpu_bool",
+    "test_quick_unfold_copy_xpu_int32",
+    "test_quick_unfold_copy_xpu_int64",
+    "test_quick_unfold_copy_xpu_int16",
+    "test_quick_unfold_xpu_bool",
+    "test_quick_unfold_xpu_int64",
+    "test_quick_unfold_xpu_int32",
+    "test_comprehensive_unfold_xpu_bool",
+    "test_quick_unfold_xpu_int16",
+    "test_quick_unfold_copy_xpu_int8",
+    "test_quick_unfold_copy_xpu_uint8",
+    "test_quick_unfold_xpu_int8",
+    "test_quick_unfold_xpu_uint8",
+    "test_comprehensive_unfold_copy_xpu_float16",
+    "test_comprehensive_unfold_xpu_float16",
+    "test_comprehensive_unfold_xpu_bfloat16",
+    "test_quick_nn_functional_unfold_xpu_bfloat16",
+    "test_quick_nn_functional_unfold_xpu_float16",
+    "test_quick_unfold_copy_xpu_float16",
+    "test_quick_unfold_copy_xpu_bfloat16",
+    "test_quick_unfold_xpu_bfloat16",
+    "test_quick_unfold_xpu_float16",
+    "test_quick_core_backward_nn_functional_unfold_xpu_float64",
+    "test_quick_core_backward_unfold_copy_xpu_float64",
 )
 res += launch_test("test_decomp_xpu.py", exe_list=execute_list)
 
 # test_torch
 execute_list = (
     "test_pin_memory",
+    "test_unfold_scalars_xpu",
+    "test_unfold_neg_dim",
+    "test_unfold_all_devices_and_dtypes_xpu",
 )
 res += launch_test("test_torch_xpu.py", exe_list=execute_list)
```

Comment:
We have enabled test_torch_xpu in run_test_with_skip.py. Please check you cases if they are covered there. If so, please remove all of test_torch_xpu form run_test_with_only.py